apis/Google.Cloud.PubSub.V1/Google.Cloud.PubSub.V1/HiPerfSubscriber.cs

```diff
@@ -94,7 +94,8 @@ internal Settings(Settings other)
             public TimeSpan? StreamAckDeadline { get; set; }
 
             /// <summary>
-            /// Duration before message expiration.
+            /// Duration before <see cref="StreamAckDeadline"/> at which the message ACK deadline
+            /// is automatically extended.
```

Comment:
Ah, so if this is 5 seconds and the ack deadline is 20 seconds, then in 15 seconds we extend it, yes? (So this is to give enough time for the extension to actually happen.)